docs/content/preview/integrations/rabbitmq.md

```diff
@@ -0,0 +1,474 @@
+
+    ```java
+    public class App {
+      public static void main(String[] args) {
+        System.out.println("Starting embedded application to run DBZ Embedded engine with Yugabyte");
+        CmdLineOpts configuration = CmdLineOpts.createFromArgs(args);
+        try {
+          EngineRunner engineRunner = new EngineRunner(configuration);
+          engineRunner.run();
+        } catch (Exception e) {
+          System.out.println("Exception while trying to run the engine: " + e);
+          System.exit(-1);
+        }
+      }
+    }
+    ```
+
+1. Create a configuration file `CmdLineOpts.java` in the `src` folder, where you can store the connection properties to connect to YugabyteDB. Add the following code to it. Set the "streamId" to the stream ID generated in step 1:
+
+    ```java
+
+    package com.dbzapp;
+    import java.util.Properties;
+    import org.apache.commons.cli.CommandLine;
+    import org.apache.commons.cli.CommandLineParser;
+    import org.apache.commons.cli.DefaultParser;
+    import org.apache.commons.cli.Options;
+    /**
+     * Helper class to parse the command line options.
+     *
+     * @author Sumukh Phalgaonkar, Vaibhav Kushwaha (vkushwaha@yugabyte.com)
+     */
+    public class CmdLineOpts {
+      private final String connectorClass = "io.debezium.connector.yugabytedb.YugabyteDBConnector";
+      public String masterAddresses;
+      public String hostname;
+      public String databasePort = "5433";
+      public String streamId;
+      public String tableIncludeList;
+      public String databaseName = "yugabyte";
+      public String databasePassword = "yugabyte";
+      public String databaseUser = "yugabyte";
+      public String snapshotMode = "never";
+
+      public static CmdLineOpts createFromArgs(String[] args) {
+        Options options = new Options();
+        options.addOption("master_addresses", true, "Addresses of the master process");
+        options.addOption("stream_id", true, "DB stream ID");
+        options.addOption("table_include_list", true, "The table list to poll for in the form"
+                          + " <schemaName>.<tableName>");
+        CommandLineParser parser = new DefaultParser();
+        CommandLine commandLine = null;
+        try {
+          commandLine = parser.parse(options, args);
+        } catch (Exception e) {
+          System.out.println("Exception while parsing arguments: " + e);
+          System.exit(-1);
+        }
+        CmdLineOpts configuration = new CmdLineOpts();
+        configuration.initialize(commandLine);
+        return configuration;
+      }
+      private void initialize(CommandLine commandLine) {
+          masterAddresses = "127.0.0.1:7100";
+          String[] nodes = masterAddresses.split(",");
+          hostname = nodes[0].split(":")[0];
+
+          streamId = "962c322b60af4b9abd0b195fc6e3f045";
```

Comment:
Please change this stream id to the output of the stream id generated in step 1. So to this 
```
CDC Stream ID: 31dd4440caca46038ba4a9365bb89d36
```